### ISSUE TITLE
Two "Insert bar before selection" icons on Layout palette, one works, the other does "Staff type change"

### ISSUE THREAD

**Issue by dulciepercy** [musescore/MuseScore#20722]
### Issue type

UI bug (incorrect info or interface appearance)

### Bug description

The "Layout Palette" has two icons for "Insert bar before selection" .  One appears by default, the second one is added by clicking "more".  The icon that shows by default doesn't work correctly - it does the same as the "Staff type change" icon.

The icon you add after clicking "more" behaves correctly.

### Steps to reproduce

1. Load fresh install of 4.2.0
2. Add the second "insert one measure before selection" icon"  by clicking "more" on layout palette
3. The two icons behave differently

### MuseScore Version

4.2.0

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Windows 11

### Additional context

Nightly from before release 4.2.0.23460505 only has one icon which works correctly.

Just noticed it's "insert measure" in the nightly, and "insert bar" in the release.

**Comment by MarcSabatella** [comment]
I'm not seeing anything like this; is it possible your palette got customized?  What happens if you click the "..." then reset the palette?

The measure vs bar difference sounds like US vs GB English; that much could just be that your preferences are set differently for release vs development builds.

**Comment by dulciepercy** [comment]
Yes reset palette and revert to factory settings fixes it.  Another commenter in the thread has the same issue (that makes three of us) so I guess it's still a potential bug, albeit trivial.

I tried changing from US to GB English (and Spanish) and it didn't make a difference re. bar and measure.  Started a thread in Documentation Forum for this.

**Comment by MarcSabatella** [comment]
Regarding the other users who are experiencing this, it would be good to see a pointer to those reports; I haven't seen others.  It's certainly possible there is a bug causing it, but we'd need know the exact conditions that lead to it, since it clearly doesn't happen for most.  So maybe in that other thread you can see if you can find the common element, then come back here and update the description with more information.

**Comment by cadiz2** [comment]
Post concerned in forum: https://musescore.org/en/node/358321
